src/map/atcommand.c

```diff
@@ -6011,13 +6023,14 @@ ACMD_FUNC(changeleader)
 {
 	nullpo_retr(-1, sd);
 
-	if( !message[0] )
-	{
+	memset(atcmd_player_name, '\0', sizeof(atcmd_player_name));
+
+	if (!message || !*message || sscanf(message, "%23[^\n]", atcmd_player_name) < 1) {
 		clif_displaymessage(fd, msg_txt(sd,1185)); // Usage: @changeleader <party_member_name>
 		return -1;
 	}
 
-	if (party_changeleader(sd, map_nick2sd((char *) message,false),NULL))
+	if (party_changeleader(sd, map_nick2sd(atcmd_player_name,false),NULL))
 		return 0;
 	return -1;
```

Comment:
Is it really okay to return -1 here? Change guild leader returns 0 and does not care about the return value.
Also you can't really know if the inter-server call succeeds or not.